rollup.config.js

```diff
@@ -7,10 +7,10 @@ import pkg from './package.json';
 export default [
   // Brwoser-friendly UMD build
   {
-    input: 'src/main.js',
+    input: 'src/index.js',
```

Comment:
Is this a standard?

Reply:
It's pretty standard to have index.js exporting all public methods

Here is mainly because I wanted to expose some other methods from legacy & utils